code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -209,12 +209,12 @@
 
 /obj/item/weapon/storage/box/flashes/New()
 	..()
-	new /obj/item/device/assembly/flash/handheld(src)
```

Comment:
Would be nice if this was a for loop. That means easier to see how many items and easier to change to a different amount